<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-9JV455Q" name="ModulesTests.IBMScoreTests" tests="15" failures="1" timestamp="28 Feb 2017 09:51:37 GMT" time="1531.812" errors="1">
  <testcase name="GPJSCORECostSavingRecordSubmissionTest" time="60.504" classname="ModulesTests.IBMScoreTests"/>
  <testcase name="SCORE_Date_Validation_QuarterTest" time="32.140" classname="ModulesTests.IBMScoreTests"/>
  <testcase name="Field_Officer_FunctionalityTest" time="154.673" classname="ModulesTests.IBMScoreTests"/>
  <testcase name="JDE_Project_FunctionalityTest" time="49.520" classname="ModulesTests.IBMScoreTests"/>
  <testcase name="CostCalculationInUSDTest" time="35.287" classname="ModulesTests.IBMScoreTests">
    <failure type="java.lang.AssertionError" message=" expected [22.134] but found [22.135]">
      <![CDATA[java.lang.AssertionError:  expected [22.134] but found [22.135]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at Modules.IBMScore.CostCalculationInUSD(IBMScore.java:437)
	at ModulesTests.IBMScoreTests.CostCalculationInUSDTest(IBMScoreTests.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.TestNG.privateMain(TestNG.java:1355)
	at org.testng.TestNG.main(TestNG.java:1324)
]]>
    </failure>
  </testcase> <!-- CostCalculationInUSDTest -->
  <testcase name="Regional_Officer_FunctionalityTest" time="214.220" classname="ModulesTests.IBMScoreTests"/>
  <testcase name="Entry_Date_Validation_QuarterTest" time="33.708" classname="ModulesTests.IBMScoreTests"/>
  <testcase name="Verify_Draft_SCORE_All_Aprroval_For_RegionalUserTest" time="173.093" classname="ModulesTests.IBMScoreTests"/>
  <testcase name="Estimated_and_Negotiated_ValidationTest" time="35.424" classname="ModulesTests.IBMScoreTests"/>
  <testcase name="ClientCodeFunctionalityTest" time="55.172" classname="ModulesTests.IBMScoreTests"/>
  <testcase name="Corporate_Approval_Not_Deleted_By_FieldandRegionalUsersTest" time="54.465" classname="ModulesTests.IBMScoreTests"/>
  <testcase name="Only_RecordOwner_Allowed_To_Submit_ScoreTest" time="64.572" classname="ModulesTests.IBMScoreTests"/>
  <testcase name="Summition_Of_SavingsInUSD_OnApprovalPageTest" time="347.146" classname="ModulesTests.IBMScoreTests">
    <error type="org.openqa.selenium.NoSuchElementException" message="no such element: Unable to locate element: {&quot;method&quot;:&quot;id&quot;,&quot;selector&quot;:&quot;ext-gen11&quot;}
  (Session info: chrome=56.0.2924.87)
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 10.0.14393 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 20.17 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: &#039;unknown&#039;, revision: &#039;c7b525d&#039;, time: &#039;2016-09-01 14:52:30 -0700&#039;
System info: host: &#039;DESKTOP-9JV455Q&#039;, ip: &#039;10.1.0.131&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_111&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, mobileEmulationEnabled=false, networkConnectionEnabled=false, chrome={chromedriverVersion=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9), userDataDir=C:\Users\DELL\AppData\Local\Temp\scoped_dir1736_11966}, takesHeapSnapshot=true, pageLoadStrategy=normal, databaseEnabled=false, handlesAlerts=true, hasTouchScreen=false, version=56.0.2924.87, platform=XP, browserConnectionEnabled=false, nativeEvents=true, acceptSslCerts=true, locationContextEnabled=true, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true, unexpectedAlertBehaviour=}]
Session ID: a37903c8d95c15e8ce55d9ec2f13617d
*** Element info: {Using=id, value=ext-gen11}">
      <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"id","selector":"ext-gen11"}
  (Session info: chrome=56.0.2924.87)
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 10.0.14393 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 20.17 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: 'unknown', revision: 'c7b525d', time: '2016-09-01 14:52:30 -0700'
System info: host: 'DESKTOP-9JV455Q', ip: '10.1.0.131', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_111'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, mobileEmulationEnabled=false, networkConnectionEnabled=false, chrome={chromedriverVersion=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9), userDataDir=C:\Users\DELL\AppData\Local\Temp\scoped_dir1736_11966}, takesHeapSnapshot=true, pageLoadStrategy=normal, databaseEnabled=false, handlesAlerts=true, hasTouchScreen=false, version=56.0.2924.87, platform=XP, browserConnectionEnabled=false, nativeEvents=true, acceptSslCerts=true, locationContextEnabled=true, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true, unexpectedAlertBehaviour=}]
Session ID: a37903c8d95c15e8ce55d9ec2f13617d
*** Element info: {Using=id, value=ext-gen11}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:631)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:369)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementById(RemoteWebDriver.java:418)
	at org.openqa.selenium.By$ById.findElement(By.java:218)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:361)
	at Modules.IBMScore.Summition_Of_SavingsInUSD_OnApprovalPage(IBMScore.java:1213)
	at ModulesTests.IBMScoreTests.Summition_Of_SavingsInUSD_OnApprovalPageTest(IBMScoreTests.java:120)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.TestNG.privateMain(TestNG.java:1355)
	at org.testng.TestNG.main(TestNG.java:1324)
]]>
    </error>
  </testcase> <!-- Summition_Of_SavingsInUSD_OnApprovalPageTest -->
  <testcase name="otherClientCodeFunctionalityTest" time="37.606" classname="ModulesTests.IBMScoreTests"/>
  <testcase name="Corporate_Officer_FunctionalityTest" time="184.282" classname="ModulesTests.IBMScoreTests"/>
</testsuite> <!-- ModulesTests.IBMScoreTests -->
